src/IHaskell/Flags.hs

```diff
@@ -70,8 +71,8 @@ parseFlags flags =
   in case modeIndex of
     Nothing ->
       -- Treat no mode as 'console'.
-      if "--help" `elem` flags
-        then Left $ showText (Wrap 100) $ helpText [] HelpFormatAll ihaskellArgs
+      if "--version" `elem` flags || "-V" `elem` flags
+        then Left VERSION_ipython_kernel
```

Comment:
Removing <code>CPP</code> gives this error

    Building ihaskell-0.8.3.0...
    Preprocessing library ihaskell-0.8.3.0...
    [ 1 of 25] Compiling Paths_ihaskell   ( dist/build/autogen/Paths_ihaskell.hs, dist/build/Paths_ihaskell.o )
    [ 2 of 25] Compiling IHaskellPrelude  ( src/IHaskellPrelude.hs, dist/build/IHaskellPrelude.o )
    [ 3 of 25] Compiling IHaskell.Types   ( src/IHaskell/Types.hs, dist/build/IHaskell/Types.o )
    [ 4 of 25] Compiling StringUtils      ( src/StringUtils.hs, dist/build/StringUtils.o )
    [ 5 of 25] Compiling IHaskell.Eval.Util ( src/IHaskell/Eval/Util.hs, dist/build/IHaskell/Eval/Util.o )
    [ 6 of 25] Compiling IHaskell.Flags   ( src/IHaskell/Flags.hs, dist/build/IHaskell/Flags.o )

    src/IHaskell/Flags.hs:75:19:
        Not in scope: data constructor ‘VERSION_ipython_kernel’